model/dyn_core.F90

```diff
@@ -1351,7 +1352,11 @@ subroutine dyn_core(npx, npy, npz, ng, sphum, nq, bdt, n_map, n_split, zvir, cp,
                                        isd, ied, jsd, jed,      &
                                        reg_bc_update_time,it )
 
-         call mpp_update_domains(u, v, domain, gridtype=DGRID_NE)
+!         call mpp_update_domains(u, v, domain, gridtype=DGRID_NE)
```

Comment:
Same comment as above - I would recommend deleting the old code as opposed to commenting it out

Reply:
Could you delete this commented code?